server/test/services/applications/ProgramAdminApplicationServiceTest.java

```diff
@@ -105,4 +164,192 @@ public void getNote_multipleNotes_findsLatest() throws Exception {
     // Execute, verify.
     assertThat(service.getNote(application)).contains(note);
   }
+
+  @Test
+  public void setStatus_sendsEmail() throws Exception {
+    String userEmail = "user@email.com";
+    SimpleEmail simpleEmail = Mockito.mock(SimpleEmail.class);
+    String programName = "some-program";
+    service =
+        new ProgramAdminApplicationService(
+            instanceOf(ApplicantService.class),
+            instanceOf(ApplicationRepository.class),
+            instanceOf(ApplicationEventRepository.class),
+            instanceOf(Config.class),
+            simpleEmail);
+
+    ProgramDefinition program =
+        ProgramBuilder.newActiveProgram("some-program")
```

Comment:
I'd update this test to use different names for the internal / external representation of the program name and assert that the subject contains the external representation (see comment in the corresponding service definition for the type of bug I'm thinking of).